pytest.ini

```diff
@@ -1,22 +1,9 @@
 # spell-checker:ignore filterwarnings norecursedirs optionflags
 [pytest]
 addopts =
-    # `pytest-xdist`:
-    -n auto
```

Comment:
Hooray! Now I don't have to comment this out when I'm running a single test in pycharm.

I still have to comment out `--pyargs` and `test ansiblelint.rules` so I can select a single test to run, but this is still much better.